Library/include/CSP/Systems/Quota/QuotaSystem.h

```diff
@@ -0,0 +1,101 @@
+/*
+* Copyright 2023 Magnopus LLC
+
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include "CSP/Systems/Quota/Quota.h"
+#include "CSP/Systems/SystemBase.h"
+
+
+namespace csp::services
+{
+
+class ApiBase;
+
+}
+
+
+namespace csp::web
+{
+
+class WebClient;
+
+}
+
+namespace csp::systems
+{
+/// @ingroup GraphQL System
+/// @brief Public facing system that allows interfacing with Magnopus Connect Services' Quota Server.
+/// Offers methods for receiving Quota Queries.
+class CSP_API CSP_NO_DISPOSE QuotaSystem : public SystemBase
+{
+	/** @cond DO_NOT_DOCUMENT */
+	friend class SystemsManager;
+	/** @endcond */
+
+public:
+	~QuotaSystem();
+
+	/// @brief Get the total number of Spaces own by the current user and their tier Space limit
+	/// @param Callback FeatureProgressCallback : callback when asynchronous task finishes
+	CSP_ASYNC_RESULT void GetTotalSpacesOwnedByUser(FeatureLimitCallback Callback);
+
+	/// @brief Gets total number of user inside of a space
```

Comment:
This api doesn't return the tier's limit of users in a space as well ? Does only that first api call return current and total allowed ?

Reply:
sorted